netlify.toml

```diff
@@ -0,0 +1,25 @@
+# https://docs.netlify.com/configure-builds/file-based-configuration/#build-settings
+[build]
+  base = ""
+  publish = "packages/website/build"
+  command = "NX_VERBOSE_LOGGING=true yarn patch-package && yarn nx build website"
```

Comment:
Non-blocking: We could potentially remove the opt-in verbose logging, it was only needed while troubleshooting something a long while back